WalkAndTravel/ClientApp/src/components/Sidebar.js

```diff
@@ -0,0 +1,59 @@
+﻿import React from 'react';
+import { slide as Menu } from 'react-burger-menu';
+import { Button } from 'react-bootstrap';
+import { Form } from 'react-bootstrap';
+import { Dropdown } from 'react-bootstrap';
+import { DropdownButton } from 'react-bootstrap';
+import { ButtonGroup } from 'react-bootstrap';
+import './Sidebar.css';
+
+export default props => {
+    return (
+        <Menu right width={450}>
```

Comment:
Maybe it is possible to remove overlay from sidebar or to find another solution, because we would want to interact with map while sidebar is turned on.